src/scripts/test.sh

```diff
@@ -5,39 +5,64 @@ cd "$(dirname "${BASH_SOURCE[0]}")/../.."
 
 export ORY_SDK_URL=https://playground.projects.oryapis.com
 
+# ensure ports are free
+npx kill-port --port 3001,3002,3003,3004,3005,3006,3007,4002,4003,4004,4005,4006,4007,4008
```

Comment:
a safety measure to free up any ports for the tests. on the ci i don't think it matters too much, but on local it's annoying since the processes are run in the background.